.gitlab-ci.yml

```diff
@@ -0,0 +1,23 @@
+push dev:
+  tags:
+    - meao
+  only: dev
+  script:
+    - bin/build.sh dev
+    - bin/push-gh-pages.sh
+
+push stage:
+  tags:
+    - meao
+  only: master
+  script:
+    - bin/build.sh stage
+    - "cd release && aws s3 sync . s3://awebpodcast-stage --acl public-read --delete --profile awebpodcast"
```

Comment:
we don't need the `--profile awebpodcast` anymore now that we're using an IAM role on the gitlab runner.